posthog/temporal/data_imports/pipelines/pipeline/utils.py

```diff
@@ -398,6 +401,29 @@ def _update_last_synced_at_sync(schema: ExternalDataSchema, job: ExternalDataJob
     schema.save()
 
 
+def _notify_revenue_analytics_that_sync_has_completed(schema: ExternalDataSchema, logger: FilteringBoundLogger) -> None:
+    try:
+        sync_took_more_than_5_minutes = schema.source.created_at < datetime.datetime.now() - datetime.timedelta(
+            minutes=5
+        )
+
+        if (
+            schema.name == STRIPE_CHARGE_RESOURCE_NAME
+            and schema.source.source_type == ExternalDataSource.Type.STRIPE
+            and schema.source.revenue_analytics_enabled
+            and sync_took_more_than_5_minutes
+            and not schema.team.revenue_analytics_config.notified_first_sync
+        ):
+            if not settings.TEST:
+                send_revenue_analytics_sync_complete_email.delay(schema.team.pk, schema.source.pk)
+            schema.team.revenue_analytics_config.notified_first_sync = True
+            schema.team.revenue_analytics_config.save()
+    except Exception as e:
+        # Silently fail, we don't want this to crash the pipeline
+        # Sending an email is not critical to the pipeline
+        logger.exception(f"Error notifying revenue analytics that sync has completed: {e}")
```

Comment:
You may wanna use `capture_exception(e)` here too so that these errors come through to the error tracking product